docs/source/driver-framework/drivers-overview.rst

```diff
@@ -74,10 +74,10 @@ Connectivity of the platform follows the following paradigm:
 
 * Platform agents (including the Platform Driver and Actuator), subsystems, and user agents communicate with the message
   bus via a publish/subscribe system.
-* Agents can communicate "directly" to each other via JSONRPC calls. JSONRPC calls use the VOLTTRON message bus router
-  to "direct" messages to an intended recipient. RPC calls from an agent specify a function for the recipient to
-  perform including input parameters; the response to the sender should contain the value output by the specified
-  function.
+* Agents can communicate "directly" to each other via JSONRPC (RPC). A JSONRPC call uses the VOLTTRON message bus router
+  to "direct" messages to an intended recipient. For example, suppose Agent B has a function called `do_task(x)` which
+  takes an `x` as an input parameter and returns an output. Using an RPC call, Agent A can specify the input parameter and
+  call Agent B's `do_task(x)` function. Agent B would receive the call, execute the task, and return the required output to Agent A.
```

Comment:
seems like if you are going to add this many words a diagram of what an rpc call would be good?

Reply:
Will do this in another PR.